nixos/doc/manual/configuration/profiles/docker-container.xml

```diff
@@ -0,0 +1,15 @@
+
+<section xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         version="5.0"
+         xml:id="sec-profile-docker-container">
+ <title>Docker Container</title>
+ <para>
+  This is the profile from which the Docker images are generated. It prepares a
+  working system by importing the <link linkend="sec-profile-minimal">Minimal</link> and
+  <link linkend="sec-profile-clone-config">Clone Config</link> profiles, and setting appropiate
```

Comment:
```suggestion
  <link linkend="sec-profile-clone-config">Clone Config</link> profiles, and setting appropriate
```

Reply:
Silly ninja R's that my brain can't hear. Thanks!